client/src/providers/HealthProvider.tsx

```diff
@@ -274,7 +274,7 @@ export const HealthProvider: React.FC<Props> = ({ children, ...props }) => {
       const signer = provider.getSigner();
       const contract = await fetchContract(signer);
       const doctors = await contract.getAllDoctorsForProvider(
-        '0xB3CC507e752Dcc3DA1cEf955B58e97Ae77160103',
+        '0x29954F7DceDC3c5d7A44A3a69Abd101Acb62b98B',
```

Comment:
if these values do not exist right now, pull them to a central location for easy tracking

Reply:
Hardcoded the address for now, but would be replaced with the ticket: https://codebrigadesjsu.atlassian.net/browse/HEAL-97